pvlib/atmosphere.py

```diff
@@ -698,3 +698,153 @@ def angstrom_alpha(aod1, lambda1, aod2, lambda2):
     pvlib.atmosphere.angstrom_aod_at_lambda
     """
     return - np.log(aod1 / aod2) / np.log(lambda1 / lambda2)
+
+
+def caballero_spectral_correction(airmass_absolute, aod500, pw,
+                                  module_type=None, coefficients=None,
+                                  aod500_ref=0.084, pw_ref=1.42):
```

Comment:
I would personally take out the last two parameters.

Reply:
Hi @adriesse, I understand that the last 2 parameters are needed, they are constant values, so we must define them as inputs variables or inside the function.